public/app/features/alerting/unified/hooks/useAbilities.ts

```diff
@@ -231,9 +239,21 @@ export function useAllAlertmanagerAbilities(): Abilities<AlertmanagerAction> {
       // TODO: Move this into the permissions config and generalise that code to allow for an array of permissions
       isGrafanaFlavoredAlertmanager ? AccessControlAction.AlertingReceiversCreate : null
     ),
-    [AlertmanagerAction.ViewContactPoint]: toAbility(AlwaysSupported, notificationsPermissions.read),
-    [AlertmanagerAction.UpdateContactPoint]: toAbility(hasConfigurationAPI, notificationsPermissions.update),
-    [AlertmanagerAction.DeleteContactPoint]: toAbility(hasConfigurationAPI, notificationsPermissions.delete),
+    [AlertmanagerAction.ViewContactPoint]: toAbility(
+      AlwaysSupported,
+      notificationsPermissions.read,
```

Comment:
Updating some missed permissions, but this is also in https://github.com/grafana/grafana/pull/94898/files#diff-9d113f692017ab6eb1395cde65f24fe40509cb6018725483c4d6a3faf42777a3L234 so I'm happy to remove it from here (but it all needs to go in eventually)

Reply:
I've just approved the templates RBAC PR ✅